packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -1075,6 +1075,49 @@ const fn = new lambda.Function(this, 'MyLambda', {
 });
 ```
 
+## IPv6 support
+
+You can configure IPv6 connectivity for lambda function by setting `Ipv6AllowedForDualStack` to true. 
+It allows Lambda functions to specify whether the IPv6 traffic should be allowed when using dual-stack VPCs. 
+To access IPv6 network using Lambda, Dual-stack VPC is required. Using dual-stack VPC a function communicates with subnet over either of IPv4 or IPv6.
+
+```ts
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+
+const natProvider = ec2.NatProvider.gateway();
+
+// create dual-stack VPC
+const vpc = new ec2.Vpc(this, 'DualStackProtocolVpc', {
+  ipProtocol: ec2.IpProtocol.DUAL_STACK,
+  subnetConfiguration: [
+    {
+      name: 'Pub1',
```

Comment:
nit: can we use `Pub1` and `Private1` in this case? They're more clear to customers.

Reply:
Done